### PR TITLE
Add 4.4 weapons and outfits

### PR DESCRIPTION
**Pull request by majkus522**

### CODE REVIEW

File: assets/data/characters/ganyu/en.json [hunk 1]
```diff
@@ -179,5 +179,15 @@
...
-  "weapon_type": "BOW"
+  "weapon_type": "BOW",
+  "outfits": [
+    {
+      "type": "Themed",
+      "name": "Twilight Blossom",
+      "description": "A light and graceful festive outfit accented mainly in black nightshade and cerulean blue. Its hair accessory is as radiant as the Qingxin — a pairing that Ganyu is most fond of.",
+      "rarity": 5,
+      "price": 1680,
+      "image": "outfit-twilight-blossom"
```
Comment (M4GICB) [inline assets/data/characters/ganyu/en.json]: [rarity should be 4](https://genshin-impact.fandom.com/wiki/Twilight_Blossom)

File: assets/data/characters/shenhe/en.json [hunk 2]
```diff
@@ -89,5 +89,15 @@
...
-  "weapon_type": "POLEARM"
+  "weapon_type": "POLEARM",
+  "outfits": [
+    {
+      "type": "Themed",
+      "name": "Frostflower Dew",
+      "description": "This Lantern Rite gift that Shenhe received is a set of elegant formal attire that her master, Cloud Retainer had commissioned for her. Red ropes still adorn her arms, but the bitter cold in Shenhe's heart has faded.",
+      "rarity": 5,
+      "price": 1680,
+      "image": "outfit-frostflower-dew"
```
Comment (M4GICB) [inline assets/data/characters/shenhe/en.json]: [rarity should be 4](https://genshin-impact.fandom.com/wiki/Frostflower_Dew)

**Review by M4GICB — CHANGES_REQUESTED** [review]
needs a couple edits on rarities

**Comment by majkus522** [comment]
Fixed

**Review by M4GICB — APPROVED** [review]

**Pull request merged**